### PR TITLE
app-text/aha: new package, add 0.5.1

### PR DESCRIPTION
**Pull request by zpuskas**
- app-text/aha: new package, add 0.5.1
- kde-plasma/kinfocenter: Add firmware security optfeatures

### CODE REVIEW

File: app-text/aha/aha-0.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Converts ANSI escape sequences of a unix terminal to HTML code."
```
Comment (juippis) [inline app-text/aha/aha-0.5.1.ebuild:6]:
```suggestion
DESCRIPTION="Converts ANSI escape sequences of a unix terminal to HTML code"
```

File: app-text/aha/aha-0.5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+HOMEPAGE="https://github.com/theZiz/aha"
+SRC_URI="https://github.com/theZiz/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2+ MPL-1.1"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+BDEPEND=""
```
Comment (juippis) [inline app-text/aha/aha-0.5.1.ebuild:16]:
```suggestion
```

File: app-text/aha/aha-0.5.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Converts ANSI escape sequences of a unix terminal to HTML code."
+HOMEPAGE="https://github.com/theZiz/aha"
+SRC_URI="https://github.com/theZiz/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (juippis) [inline app-text/aha/aha-0.5.1.ebuild:8]:
```suggestion
SRC_URI="https://github.com/theZiz/aha/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
```
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI
(and it's also shorter)

**Review by juippis — COMMENTED** [review]
Also seeing as 0.5.1 was released in 2020 but there has been commits afterwards, maybe do a snapshot release for the latest commit?

**Comment by zpuskas** [comment]
> Also seeing as 0.5.1 was released in 2020 but there has been commits afterwards, maybe do a snapshot release for the latest commit?

I'll ask upstream to make a new release. If that does not happen, I'll make a snapshot version.

**Comment by zpuskas** [comment]
I've submitted a new release request to upstream: https://github.com/theZiz/aha/issues/98

**Comment by thesamesam** [comment]
Please rebase.

**Comment by a17r** [comment]
It makes me a bit sad that we are adding something without a commit in almost a year and something like this pending https://github.com/theZiz/aha/pull/97

**Comment by zpuskas** [comment]
> It makes me a bit sad that we are adding a new package without an upstream commit in almost a year and something like this pending [theZiz/aha#97](https://github.com/theZiz/aha/pull/97)

I can port that patch to this ebuild. Unfortunately kinfocenter relies on this package to display firmware related information, without it the info panel is broken. This probably should be solved by KDE upstream (should we open a bug/feature request to replace this dependency?), but until such times I'd prefer we fix desktop user experience.

**Comment by zpuskas** [comment]
I've applied the squashed commits as a patch from https://github.com/theZiz/aha/pull/97.

**Review by juippis — APPROVED** [review]
I think I would still prefer a snapshot version. @a17r what do you say?

**Review by a17r — APPROVED** [review]
Let's add this for now - it is what other distributions seem to use as well.

@zpuskas You've squashed the wrong commits. I'll fix that for you on merge.

**Comment by zpuskas** [comment]
> You've squashed the wrong commits. I'll fix that for you on merge.

Ah, apologies, I did not notice that. Thanks for the fix and the merge.

**Pull request closed without merging**